crates/forge/src/pretty_printing.rs

```diff
@@ -5,6 +5,11 @@ use forge_runner::{test_case_summary::AnyTestCaseSummary, test_crate_summary::Te
 use starknet_api::block::BlockNumber;
 use std::collections::HashMap;
 
+pub fn print_warning(error: &Error) {
+    let warning_tag = style("WARNING").yellow();
```

Comment:
```suggestion
    let warning_tag = style("WARNING").color256(11);
```
It is yellow too, but different from the one we use for `[IGNORE]`

Reply:
Done